project.md

```diff
@@ -11,41 +11,59 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I came up with this idea because I was in my Economics class and I was recalling how bothersome it has been in the past to have to find different calculator tools or open up a Google sheet just for one or two calculations for prior Economics assignments. While I may not have to use these tools in the future, I can appreciate how useful they may be for others in my position who want to just quickly perform a couple calculations for a homework set without having to open a google sheet and then find google sheets documentation on what order the arguments go into the formula. Additionally, I am excited to be able to apply my background in Computer Science to create a tool for people studying another field.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Economic and Finance Formula Calculations
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+A user would interact with this language in a similar format to what is the standard of Google/Excel sheets, which is like calling functions with multiple arguments in general purpose programming languages (i.e formula(arg1, arg2, ...) ). This is the right way to interact with the problem domain because many of the users are likely not coming from programming backgrounds, or at least this is not an expected requisite, so in asking them to write code, I am hoping to keep their experience as similar to that which they are used to in used the existing software, avoiding many of the technical concepts we use as programmers.
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When a program runs, it will check for function calls and their respective arguments, telling users if they have provided invalid arguments (i.e. argument which must be positive was given a negative value, 2 arguments given when 1 was expected, etc.) This would also be useful for users because in my experience, the error handling by software like Google and Excel is minimal and not very useful for determining what was wrong with the input. This will require more testing to ensure that the errors provided are more clear for the users who are not expected to have backgrounds in coding. Some examples of error handling I have considered before are: "division by zero error", "Invalid value for X", and "Not enough arguments. This function expects the following arguments: X, Y, and Z".
```

Comment:
I think this will be very valuable to the user as well. Also make sure that your errors for the user are understandable by non coders.